esp/scm/ws_config2.ecm

```diff
@@ -329,6 +330,7 @@ ESPrequest UnlockSessionRequest
 {
     string SessionId;
     string SessionLockKey;
+    bool   FailIfModified(true);
```

Comment:
perhaps too picky, but fail seems to imply something greater than rejecting the request...
how about rejectIfModified?

Reply:
Changed.